app/model/sdl/VehicleInfoModel.js

```diff
@@ -331,6 +332,16 @@ SDL.SDLVehicleInfoModel = Em.Object.create(
          'width': 800,
          'height': 480
       },
+      'seatStatus':{
+        'seatLocation': true,
+        'conditionActive':true
+      },
```

Comment:
@Ypostolov you don't need to declare this structure, because `SeatStatus` is an element of `SeatOccupancy` array. Please remove this

Reply:
@AKalinich-Luxoft 
Fixed in 6db411a .
Please check.